en/releases/main.md

```diff
@@ -48,6 +48,12 @@ TBD ...
   Gazebo additional supports drag-and-drop via the resource spawner in Gazebo GUI.
   Added to PX4 in [PX4-Autopilot#22467](https://github.com/PX4/PX4-Autopilot/pull/22467).
 
+### Ethernet
+
+- [Ethernet](../advanced_config/ethernet_setup.md): The default static IP address (used as a fallback for DHCP) has changed from `192.168.0.3` to `10.41.10.2`.
```

Comment:
@bkueng I added this release note. I'm trying to do this for all changes that touch docs FYI, because I think it is useful, but also so that when we get to the next release we're not scabbling to remember what we did and why. Appreciate it if you could check this (and of course start to make adding these a habit).